src/strategy.rs

```diff
@@ -332,16 +561,17 @@ where
                         }
                         false
                     })
-                    .unwrap_or(false)
+                    .unwrap_or(false);
```

Comment:
An alternative idea is a helper trait on Bool adding .inverted(inverted) - that will allow

```
.unwrap_or(false).inverted(inverted)
```